bin/varnishd/cache/cache_director.c

```diff
@@ -80,27 +80,32 @@ VDI_Ahealth(const struct director *d)
 
 /* Resolve director --------------------------------------------------*/
 
+VCL_BACKEND
+VRT_VDI_Resolve(VRT_CTX, VCL_BACKEND d)
```

Comment:
So except for slightly different assertions, the `VRT_VDI_Resolve()` from this PR is identical to `VRT_DirectorResolve()` which I proposed in #2680 so, sure, I am all for it (and would think that we should still polish the assertions).